.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: core-sdk CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # add versions to array as we move support forward
+        node-version: [12.x]
```

Comment:
I believe we are only targeting 12 LTS for now, but I can add others if we want to test specific versions.

Reply:
The SDK needs to broader support than the CLI (especially since we're releasing the CLI as a binary). It looks like Node version 10 is still being actively patched so it seems like we should at least go back that far. @bflorian What do you think?

Reply:
I will add `10.x` in here as well. Good call out.